components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1103 @@
+
+```{figure} images/tilde_w.jpeg
+:align: center
+:width: 300 px
+
+Schematic of pseudo-velocities.
+```
+
+$\tilde{W}_{tr}$ will be introduced and defined in [](#notational-simplifications). Using this projection, we obtain:
+
+$$
+\frac{d}{dt} \int_A \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}^{\text{top}}} \, \varphi \, d\tilde{z} \, dA
+& + \int_{\partial A} \left( \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}^{\text{top}}} \varphi \, {\bf u} \, d\tilde{z} \right) \cdot {\bf n}_\perp \, dl \\
+& + \int_A \rho_0 \varphi \left[ \tilde{w}_{tr} - \tilde{ u} \right]_{\tilde{z} = \tilde{z}^{\text{top}}} dA \\
+& - \int_A \rho_0 \varphi \left[ \tilde{w}_{tr} - \tilde{ u} \right]_{\tilde{z} = \tilde{z}^{\text{bot}}} dA
+= 0
+$$ (tr-vh-separation-pseudo)
+
+This equation is structurally identical to its $z$-based counterpart but with all references to geometric height replaced by pseudo-height quantities.
+
+This formulation allows vertical mass and tracer transport to be computed directly in terms of prognostic variables, without reconstructing $\rho$ or $z$ explicitly.
+
+Similar expressions to [](#tr-vh-separation-pseudo) hold for mass and momentum:
+
+**Mass:**
+
+$$
+\frac{d}{dt} \int_A \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}^{\text{top}}} \, d\tilde{z} \, dA
+& + \int_{\partial A} \left( \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}^{\text{top}}} \, {\bf u} \, d\tilde{z} \right) \cdot {\bf n}_\perp \, dl \\
+& + \int_A \rho_0 \left[ \tilde{w}_{tr} - \tilde{ u} \right] dA \\
+& - \int_A \rho_0 \left[ \tilde{w}_{tr} - \tilde{ u} \right] dA
+= 0
+$$ (vh-mass-pseudo)
+
+**Tracer:**
+
+$$
+\frac{d}{dt} \int_A \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}_k^{\text{top}}}  \varphi \, d\tilde{z} \, dA
+& + \int_{\partial A} \left( \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}_k^{\text{top}}}  \varphi \, {\bf u} \, d\tilde{z} \right) \cdot {\bf n}_\perp \, dl & \\
+& + \int_A \left\{ \rho_0 \varphi \left[\tilde{w}_{tr} - \tilde{ u} \right] \right\}_{\tilde{z} = \tilde{z}^{\text{top}}} \, dA & \\
+& - \int_A \left\{ \rho_0 \varphi \left[\tilde{w}_{tr} - \tilde{ u} \right] \right\}_{\tilde{z} = \tilde{z}^{\text{bot}}}\, dA
+& = 0
+$$ (vh-tracer-pseudo)
+
+**Momentum:**
+
+$$
+\frac{d}{dt} \int_{A} \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}^{\text{top}}} \, {\bf u} \, d\tilde{z} \, dA
+&+
+\int_{\partial A} \left( \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}^{\text{top}}} \, {\bf u} \otimes {\bf u} \, d\tilde{z} \right) \cdot {\bf n}_\perp \, dl \\
+&+
+\int_A \rho_0 \, {\bf u} \left[ \tilde{w}_{tr} - \tilde{ u} \right]_{\tilde{z} = \tilde{z}^{\text{top}}} \, dA
+-
+\int_A \rho_0 \, {\bf u} \left[ \tilde{w}_{tr} - \tilde{ u} \right]_{\tilde{z} = \tilde{z}^{\text{bot}}} \, dA \\
+&=
+\int_A \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}^{\text{top}}} {\bf b}_{\perp} \, d\tilde{z} \, dA
```

Comment:
I don't see the definition of {\bf b}_{\perp}. Let's add it below for clarity, given that I suppose it is normal to A, not \cdot {\bf n}_\perp

Reply:
added